.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!--- These comments are hidden when you submit the pull request, --->
+<!--- so you do not need to remove them. --->
+
+<!--- Before opening a pull request, go over all the following points: --->
+<!--- * I have read the contributor guide (CONTRIBUTING.md) --->
+<!--- * My code follows the code style of this project --->
+<!--- * I added tests to cover my changes --->
+<!--- * If needed, I updated the documentation --->
+<!--- * If needed, I added translations if they are needed --->
+<!--- * If needed, I added migration files and checked for potential conflicts --->
+
+<!--- This project only accepts pull requests related to open issues. --->
+<!--- If suggesting a new feature or change, please discuss it in an issue first. --->
+<!--- If fixing a bug, there should be an issue describing it with steps to reproduce. --->
+
+<!--- Please provide a general summary of your changes in the title of the pull request --->
+
+## Description
+<!--- Describe your changes in detail --->
+
+<!--- Please link to a related issue here --->
+Related issue: #ISSUE_NUMBER
```

Comment:
Should we prefer

`Fixes #ISSUE_NUMBER` or `Closes #ISSUE_NUMBER`?

This way GitHub will automatically link PR and issue (on the right side of the UI under "development"), meaning merging the PR will close the issue.

Whereas using `Related issue: #ISSUE_NUMBER` simply links them, merging does not close the issue.

Reply:
Alright. I think it needs to be `Resolves #ISSUE_NUMBER` (no colon). And yes, once the changes reach the default branch, issues will be closed.
Changing to `Resolves`.